articles/container-instances/container-instances-update.md

```diff
@@ -58,6 +58,8 @@ Applications based on larger container images like Windows Server Core can see s
 * The IP address of a container group is typically retained between updates, but isn't guaranteed to remain the same. As long as the container group is deployed to the same underlying host, the container group retains its IP address. Although rare, there are some Azure-internal events that can cause redeployment to a different host. To mitigate this issue, we recommend using a DNS name label for your container instances.
 * Terminated or deleted container groups can't be updated. Once a container group is stopped (is in the *Terminated* state) or deleted, the group is deployed as new.
 
+> [!NOTE]
+> Update command could not work if the Azure Container Group attched to Azure Storage profile
```

Comment:
```suggestion
> [!NOTE]
> The update command might not work if the Azure Container Group is attached to an Azure Storage profile.
```

Addressed some typos... just in case this change is accepted.